src/bioclip/__main__.py

```diff
@@ -32,17 +32,32 @@ def write_results_to_file(df, format, outfile):
         raise ValueError(f"Invalid format: {format}")
 
 
+def parse_bins_csv(bins_path):
+    if not os.path.exists(bins_path):
+        raise FileNotFoundError(f"File not found: {bins_path}")
+    bin_df = pd.read_csv(bins_path, index_col=0)
+    if len(bin_df.columns) == 0:
```

Comment:
```suggestion
    if len(bin_df.columns) < 2:
```
Since it requires at least two columns?

Reply:
Since the first column is used as the index via `index_col=0` there only needs to be one "column".